src/main/java/de/muenchen/isi/domain/service/AbfrageStatusService.java

```diff
@@ -1,59 +1,274 @@
 import de.muenchen.isi.infrastructure.entity.enums.lookup.StatusAbfrage;
+import de.muenchen.isi.infrastructure.entity.enums.lookup.StatusAbfrageEvents;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHeaders;
+import org.springframework.messaging.support.MessageBuilder;
+import org.springframework.statemachine.StateContext;
+import org.springframework.statemachine.StateMachine;
+import org.springframework.statemachine.StateMachineEventResult;
+import org.springframework.statemachine.config.StateMachineFactory;
+import org.springframework.statemachine.state.State;
+import org.springframework.statemachine.support.DefaultStateMachineContext;
+import org.springframework.statemachine.support.StateMachineInterceptor;
+import org.springframework.statemachine.support.StateMachineInterceptorAdapter;
 import org.springframework.stereotype.Service;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
 
-import java.util.Objects;
 import java.util.UUID;
 
+/**
+ * Der Zustandsautomat ist in {@link  StateMachineConfiguration} konfiguriert.
+ */
 @Slf4j
 @Service
 @RequiredArgsConstructor
 public class AbfrageStatusService {
 
+    private static final String ABFRAGE_ID_HEADER = "abfrage_id";
+
     private final AbfrageService abfrageService;
 
+    private final StateMachineFactory<StatusAbfrage, StatusAbfrageEvents> stateMachineFactory;
+
+    /**
+     * Ändert den Status auf {@link StatusAbfrage#OFFEN}.
+     *
+     * @param id vom Typ {@link UUID} um die Abfrage zu finden.
+     * @throws EntityNotFoundException          falls die Abfrage nicht gefunden werden kann.
+     * @throws AbfrageStatusNotAllowedException wenn die Statusänderung nicht erlaubt ist
+     */
+    public void freigabeAbfrage(final UUID id) throws EntityNotFoundException, AbfrageStatusNotAllowedException {
+        final StateMachine<StatusAbfrage, StatusAbfrageEvents> stateMachine = this.build(id);
+        this.sendEvent(id, StatusAbfrageEvents.FREIGABE, stateMachine);
+    }
+
+
+    /**
+     * Ändert den Status auf {@link StatusAbfrage#ABBRUCH}.
+     *
+     * @param id vom Typ {@link UUID} um die Abfrage zu finden.
+     * @throws EntityNotFoundException          falls die Abfrage nicht gefunden werden kann.
+     * @throws AbfrageStatusNotAllowedException wenn die Statusänderung nicht erlaubt ist
+     */
+    public void abbrechenAbfrage(final UUID id) throws EntityNotFoundException, AbfrageStatusNotAllowedException {
+        final StateMachine<StatusAbfrage, StatusAbfrageEvents> stateMachine = this.build(id);
+        this.sendEvent(id, StatusAbfrageEvents.ABBRECHEN, stateMachine);
+    }
+
+    /**
+     * Ändert den Status auf {@link StatusAbfrage#ANGELEGT}.
+     *
+     * @param id vom Typ {@link UUID} um die Abfrage zu finden.
+     * @throws EntityNotFoundException          falls die Abfrage nicht gefunden werden kann.
```

Comment:
Anmerkung um es einheitlich mit bereits bestehenden Javadocs zu machen.

```suggestion
     * @throws EntityNotFoundException falls die Abfrage nicht gefunden wird
```